charts/dex.yaml

```diff
@@ -1,4 +1,4 @@
-image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
```

Comment:
Can be squashed with the chart commit (no real value in making another commit here, the link was just for you to look at :wink:)